OpenRA.Game/GameInformation.cs

```diff
@@ -35,6 +35,7 @@ public class GameInformation
 		/// <summary>Gets the game's duration, from the time the game started until the replay recording stopped.</summary>
 		public TimeSpan Duration { get { return EndTimeUtc > StartTimeUtc ? EndTimeUtc - StartTimeUtc : TimeSpan.Zero; } }
 		public IList<Player> Players { get; private set; }
+		public List<int> DisabledSpawnPoints;
```

Comment:
It would probably be cleaner to default this to an empty list rather than null, then we can avoid having to manually fall back to an empty list in `ReplayBrowserLogic`. What do you think?